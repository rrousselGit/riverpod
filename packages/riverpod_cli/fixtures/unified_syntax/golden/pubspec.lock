# Generated by pub
# See https://dart.dev/tools/pub/glossary#lockfile
packages:
  characters:
    dependency: transitive
    description:
      name: characters
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.2.1"
  collection:
    dependency: transitive
    description:
      name: collection
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.16.0"
  flutter:
    dependency: "direct main"
    description: flutter
    source: sdk
    version: "0.0.0"
  flutter_hooks:
    dependency: "direct main"
    description:
      name: flutter_hooks
      url: "https://pub.dartlang.org"
    source: hosted
    version: "0.18.5+1"
  flutter_riverpod:
    dependency: "direct main"
    description:
      name: flutter_riverpod
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.0.3+1"
  hooks_riverpod:
    dependency: "direct main"
    description:
      name: hooks_riverpod
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.0.3+1"
  material_color_utilities:
    dependency: transitive
    description:
      name: material_color_utilities
      url: "https://pub.dartlang.org"
    source: hosted
    version: "0.1.5"
  meta:
    dependency: transitive
    description:
      name: meta
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.8.0"
  riverpod:
    dependency: "direct main"
    description:
      name: riverpod
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.0.3+1"
  sky_engine:
    dependency: transitive
    description: flutter
    source: sdk
    version: "0.0.99"
  state_notifier:
    dependency: transitive
    description:
      name: state_notifier
      url: "https://pub.dartlang.org"
    source: hosted
    version: "0.7.2+1"
  vector_math:
    dependency: transitive
    description:
      name: vector_math
      url: "https://pub.dartlang.org"
    source: hosted
    version: "2.1.2"
sdks:
  dart: ">=2.17.0 <3.0.0"
  flutter: ">=3.0.0"
