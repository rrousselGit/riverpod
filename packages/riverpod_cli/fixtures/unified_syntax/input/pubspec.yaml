name: codemod_riverpod_test_unified_syntax
description: Test riverpod codemods
version: 0.0.1
publish_to: none

environment:
  sdk: ">=2.12.0-0 <3.0.0"

dependencies:
  flutter:
    sdk: flutter
  flutter_hooks:
  flutter_riverpod: ^0.14.0+1
  hooks_riverpod: ^0.14.0+1
  riverpod: ^0.14.0+1
