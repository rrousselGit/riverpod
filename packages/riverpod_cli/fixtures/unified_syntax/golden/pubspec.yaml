name: codemod_riverpod_test_unified_syntax_golden
description: Test riverpod codemods
version: 0.0.1
publish_to: none

environment:
  sdk: ">=2.12.0-0 <3.0.0"

dependencies:
  flutter:
    sdk: flutter
  flutter_hooks:
  flutter_riverpod: ^1.0.0-dev.10
  hooks_riverpod: ^1.0.0-dev.10
  riverpod: ^1.0.0-dev.10